js/main.js

```diff
@@ -0,0 +1,59 @@
+// This is where the magic happens
+// Written by Fredrik August Madsen-Malmo (github@fredrikaugust)
+
+var URL = "https://api.github.com/repos/mikaelbr/open-source-ideas/issues";
+
+function display_issues (flags = []) {
```

Comment:
Technically, it should bore correctly be written as

```js
flags = typeof flags === 'undefined' ? [] : flags;
```

But `flags = flags || [];` is a often used "hack"/shortcut to in many cases achieve the same effect as in javascript boolean expression return the first truthy value.

---

```js
var default1 = function (flags) {
  return typeof flags === 'undefined' ? [] : flags;
};
var default2 = function (flags) {
  return flags || [];
};

```

Here `default1` and `default2` isn't strictly the same when it comes to falsey values. See details:

```js
default1(); //> []
default1(undefined); //> []
default1(null); //> null
default1(false); //> false
default1(''); //> ''

// But with default2:
default1(); //> []
default1(undefined); //> []
default1(null); //> []
default1(false); //>  []
default1(''); //> []
```